content/posts/2017/2017-10-12-tibbles.Rmd

```diff
@@ -0,0 +1,227 @@
+---
+title: "The Trouble with Tibbles"
+author: "Theo Roe"
```

Comment:
Add a slug field. A slug is a single name that will be used to form the URL. That way if you change the title or filename, the url remains fixed.

Reply:
do I add the slug in the yaml? as in slug: "troublewithtibbles"

Reply:
Yep. Use underscores to separate words.